pages/leader_board.html

```diff
@@ -1,7 +1,7 @@
 <div ng-controller="leaderboard as lb">
     <div class="league_landing">
         <div class="league_button_history">
-            <img src="images/Trophy.png"></img> Leader Board
+            <img src="../images/Trophy.png"></img> Leader Board
```

Comment:
This change doesn't  seem to work on my local. All the other images seem to work without the leading '../'